### PR TITLE
Update PuttySessionsRegistryProvider.cs

### PR DESCRIPTION
**Pull request by BlueBlock**
## Description
remove unreachable code
- no need to test for null, PuttySessionsKey is a const with a value set on instantiation

## Motivation and Context
Unneeded code.

## How Has This Been Tested?
clean build

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Changed feature (non-breaking change which changes functionality)
- [ ] Changed feature (**breaking** change which changes functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Updated translation

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] All Tests within VisualStudio are passing
- [ ] This pull request does not target the master branch.
- [ ] I have updated the changelog file accordingly, if necessary.
- [ ] I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: mRemoteNG/Config/Putty/PuttySessionsRegistryProvider.cs [hunk 1]
```diff
@@ -24,8 +24,6 @@ public class PuttySessionsRegistryProvider : AbstractPuttySessionsProvider
 
         public override string[] GetSessionNames(bool raw = false)
         {
-            if (PuttySessionsKey == null) return null;
-
             var sessionsKey = Registry.CurrentUser.OpenSubKey(PuttySessionsKey);
             if (sessionsKey == null) return new string[] { };
```
Comment (savornicesei) [inline mRemoteNG/Config/Putty/PuttySessionsRegistryProvider.cs]: Just a small perf. improvement: it could return `Array.Empty<string>` instead of  `new string[]{}`

Comment (BlueBlock) [reply mRemoteNG/Config/Putty/PuttySessionsRegistryProvider.cs]: I'll take a look. Thanks.

Comment (Kvarkas) [reply mRemoteNG/Config/Putty/PuttySessionsRegistryProvider.cs]: I ask ChatGPT :)
image

**Review by Kvarkas — APPROVED** [review]

**Comment by BlueBlock** [comment]
The string array has been changed to Array.Empty.

Thanks.
